### PR TITLE
Update zingolib version for new scenario

### PR DESCRIPTION
**Pull request by dorianvp**
This PR updates the zingolib version to include the new `funded_transparent_mobileclient`.

### CODE REVIEW

File: rust/Cargo.toml [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-    "lib",
-    "android",
-    "zingomobile_utils"
-]
+members = ["lib", "android", "zingomobile_utils"]
 resolver = "2"
 
 [workspace.dependencies]
-zingolib = { git="https://github.com/zingolabs/zingolib", default-features=true, tag = "1.8.1", features=["test-elevation"] }
+zingolib = { git = "https://github.com/zingolabs/zingolib", default-features = true, rev = "cab1ca5b99120bee291d67f37f325715679b29fb", features = [
```
Comment (juanky201271) [inline rust/Cargo.toml]: I'm thinking about to put a specific commit... but IMO is better to put the `dev` branch... and sometimes to update the Cargo.lock after a `cargo update....` when is necessary.

File: rust/android/Cargo.toml [hunk 2]
```diff
@@ -11,8 +11,8 @@ regchest = []
 [dev-dependencies]
 zingolib = { workspace = true }
 zingomobile_utils = { path = "../zingomobile_utils" }
-regchest_utils = { git="https://github.com/zingolabs/zingo-regchest", default-features=true, branch = "dev" }
-darkside-tests = { git="https://github.com/zingolabs/zingolib", default-features=true, tag = "1.8.1" }
+regchest_utils = { git = "https://github.com/zingolabs/zingo-regchest", default-features = true, branch = "dev" }
+darkside-tests = { git = "https://github.com/zingolabs/zingolib", default-features = true, rev = "cab1ca5b99120bee291d67f37f325715679b29fb" }
```
Comment (juanky201271) [inline rust/android/Cargo.toml]: same here...

**Review by juanky201271 — CHANGES_REQUESTED** [review]

**Comment by dorianvp** [comment]
I am converting this PR back to draft. Somehow the zingolib update (both this commit and the latest one in the dev branch) broke sync for me with the following logs:
```Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=org.ZingoLabs.Zingo/.MainActivity }
 BUNDLE  ./index.js 

 LOG  Running "Zingo" with {"rootTag":11}
 LOG  false undefined BIOMETRIC_ERROR_NONE_ENROLLED
 LOG  crypto provider result true
 LOG  stop sync process. STOPPED
 LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
 LOG  stop sync process. STOPPED
 LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
 LOG  stop sync process. STOPPED
 LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
 LOG  stop sync process. STOPPED
 LOG  RPC - Update Data lock, returning *****************************
 LOG  REFRESH ----> in refresh is true
 LOG  REFRESH ----> in refresh is true
 LOG  sync finished {
  "result": "success",
  "latest_block": 2703851,
  "total_blocks_synced": 1
}
 LOG  REFRESH ----> in refresh is true
 LOG  ++++++++++ interval update 5 secs [84, 85, 224]
 LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
 LOG  stop sync process. STOPPED
 LOG  RPC - Update Data lock, returning *****************************
```
This also breaks some detox tests.

**Comment by juanky201271** [comment]
> I am converting this PR back to draft. Somehow the zingolib update (both this commit and the latest one in the dev branch) broke sync for me with the following logs:
> 
> ```
>  BUNDLE  ./index.js 
> 
>  LOG  Running "Zingo" with {"rootTag":11}
>  LOG  false undefined BIOMETRIC_ERROR_NONE_ENROLLED
>  LOG  crypto provider result true
>  LOG  stop sync process. STOPPED
>  LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
>  LOG  stop sync process. STOPPED
>  LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
>  LOG  stop sync process. STOPPED
>  LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
>  LOG  stop sync process. STOPPED
>  LOG  RPC - Update Data lock, returning *****************************
>  LOG  REFRESH ----> in refresh is true
>  LOG  REFRESH ----> in refresh is true
>  LOG  sync finished {
>   "result": "success",
>   "latest_block": 2703851,
>   "total_blocks_synced": 1
> }
>  LOG  REFRESH ----> in refresh is true
>  LOG  ++++++++++ interval update 5 secs [84, 85, 224]
>  LOG  Critical Error addresses balances notes SyntaxError: JSON Parse error: Unexpected character: s
>  LOG  stop sync process. STOPPED
>  LOG  RPC - Update Data lock, returning *****************************
> ```
> 
> This also breaks some detox tests.

this is the problem... https://github.com/zingolabs/zingolib/issues/1485 the command balance changed and now it is not a valid string JSON...

**Review by juanky201271 — APPROVED** [review]
good job.

**Pull request merged**